rpc/rpc-wrappers.go

```diff
@@ -1016,6 +1020,30 @@ func NewStorageAuthorityServer(rpc RPCServer, impl core.StorageAuthority) error
 		return
 	})
 
+	rpc.Handle(MethodCountCertificatesRange, func(req []byte) (response []byte, err error) {
+		var countReq struct {
+			Start time.Time
+			End   time.Time
+		}
+
+		err = json.Unmarshal(req, &countReq)
+		if err != nil {
+			return
+		}
+
+		count, err := impl.CountCertificatesRange(countReq.Start, countReq.End)
+		if err != nil {
+			return
+		}
+		buf := new(bytes.Buffer)
+		err = binary.Write(buf, binary.LittleEndian, count)
```

Comment:
I think doing so in those other cases is an anti-pattern, and we should probably fix it eventually. One could argue that the 33% penalty for base64-encoding the bodies is worth avoiding, but I think consistency is better.

Here, I think there's a coordination problem between the two ends of the RPC. How do we ensure that one encodes / decodes as LittleEndian vs BigEndian? I'd be much happier either (a) just using JSON.Marshal(count), which would turn it into an unambiguous decimal string, or (b) defining a CountCertificatesResponse struct with a single field.